### ISSUE TITLE
Реализовать отправку писем с восстановлением пароля

### ISSUE THREAD

**Issue by alex21072** [time-tracking-practicum/time-tracking#51]
В блоке забыли пароль, при вводе почты кнопка отправить не становится активной
При нажатии на Отправить, перекидывает на главную страницу, письмо на почту не отправляется

**Comment by Artem-Mit** [comment]
Я возьму

**Comment by Artem-Mit** [comment]
Готово. Бэкенд отвечает ошибкой.

**Comment by Artem-Mit** [comment]
![Image](https://github.com/time-tracking-practicum/time-tracking/assets/102519212/257b62f4-a282-4198-bb81-49e2b1db0829)
![Image](https://github.com/time-tracking-practicum/time-tracking/assets/102519212/2c0ebc7a-5bea-4cdb-ba54-9c95b760370c)

**Comment by apant94** [comment]
@Artem-Mit ну красота!)

**Comment by Wests007** [comment]
Да, на данный момент на сервере nginx ошибка 404. Нужен Илья @idudnikov чтобы решить эту проблему
